src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -357,4 +369,26 @@ public String getStudentStatus() {
             return Const.STUDENT_COURSE_STATUS_YET_TO_JOIN;
         }
     }
+    
+    public Date getCreated() {
+        return created;
+    }
+
+    public Date getLastUpdate() {
+        return lastUpdate;
+    }
+    
+    /**
+     * Should only be used for testing
+     **/
+    public void setCreated(Date created) {
```

Comment:
May be we can add a `_NonProduction` suffix to indicate this method is not be used in production? The comment could be missed easily.